.gitignore

```diff
@@ -18,3 +18,4 @@
 /ui/.next/
 /ui/node_modules
 /ui/public/traces/*.jsonl
+data/
```

Comment:
Which data directory is this meant to ignore?

This file is symlinked to `.dockerignore` which has slightly different semantics, so this needs to be a repo-relative path with a leading slash.